src/tdom/__init__.py

```diff
@@ -6,14 +6,7 @@
 _parsed = {}
 _listeners = []
 
-# T = (t'').__class__
-# Let's just use the imported class
-try:
-  # Temporary hack because pytest-playwright has to run under 3.13
-  # in CI due to greenlet not compiling there under 3.14.
-  from string.templatelib import Template
-except ImportError:
-  pass
+from string.templatelib import Template
```

Comment:
does this work on MicroPython port? that's also the reason it was there

Reply:
If the MicroPython flavor supports t-strings, it works. If not, then tdom isn't very useful. :)